e2e/resources/namespace.go

```diff
@@ -57,3 +61,12 @@ func RemoveNamespace(client kubernetes.Interface, name string) error {
 	}
 	return nil
 }
+
+func RandomNamespace(nameLength int) string {
```

Comment:
can we add a prefix, like `e2e-`. I am not sure which prefix is better

Reply:
This ns (namespace) will be cleaned up whether the e2e test succeeds or fails, and will not be left visible to the developers. In this situation, is it still necessary to use the prefix ?